index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-compatible" content="IE=edge">
+    <meta name="HandheldFriendly" content="true">
+       
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">   
+    
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <section class="wrapper">
+      <asside class="asside-1 asside"></asside>
+      <main class="main-view">
+        <header class="nav-bar">
+          <a href="#" class="nav-logo"> Frank</a>
+          <nav class="nav-item">
+            <a href="#">
+              <div class="menu"></div>
+              <div class="menu"></div>
+              <div class="menu"></div>
+            </a>
+          </nav>
+        </header>
+        <article class="page-content">
+
+          <!--home page-->
+          <section id="home" class="home-content">
+            <div class="circle-1">
+              <h1 class="name"><strong>Frank Tembo</strong></h1>
+              <p class="occupation"><strong>Software Developer</strong></p>
+            </div>
+            <div class="about-me descript">
+              <div class="prof">
+                <p class="prof-text">I can help you build a product, feature or website. Look through
+                some of my work and experience! If you like what you see and have a project that
+                you need coded, please don't hesitate to contact me.
+                </p>
+
+                <hr>   
+
+                <div class="lang-skills">
```

Comment:
- [ ]  Please fix the styling of Skills to match the Figma design 👇
![176657145-07593040-7678-4624-8362-3bdd8a4723b6](https://user-images.githubusercontent.com/81444338/176758530-79ea7fe5-1847-4df6-89f2-327d1b7b111b.png)

![Capturerpor](https://user-images.githubusercontent.com/81444338/176759517-0d734002-96eb-4c38-b346-169f37685617.PNG)

Your design
![Captureport](https://user-images.githubusercontent.com/81444338/176759148-fb35ac13-9836-4864-a37f-2d44c9d24d06.PNG)